### ISSUE TITLE
Unknown status 'hidden' for file ***.md

### ISSUE THREAD

**Issue by s-nt-s** [getpelican/pelican#2916]
- [x] I have read the [Filing Issues](https://docs.getpelican.com/en/latest/contribute.html#filing-issues) and subsequent “How to Get Help” sections of the documentation.
- [x] I have searched the [issues](https://github.com/getpelican/pelican/issues?q=is%3Aissue) (including closed ones) and believe that this is not a duplicate.

- **OS version and name**:  Linux Mint 19.3 Tricia
- **Python version**: Python 3.6.9
- **Pelican version**: 4.6.0
- **Link to theme**: https://github.com/s-nt-s/GSI/tree/master/themes/mini
- **Links to plugins**: https://github.com/s-nt-s/GSI/tree/master/myplugins
- **Link to your site**: http://apuntes.ml/GSI/
- **Link to your source**: https://github.com/s-nt-s/GSI
- **Link to a [Gist](https://gist.github.com/) with the contents of your settings file**: https://github.com/s-nt-s/GSI/blob/master/pelicanconf.py

## Issue

`pelican 4.6.0` doesn't recognize [`hidden` status for posts](https://docs.getpelican.com/en/latest/content.html#hidden-posts).

```
$ head -n 5 content/posts/temario/BOE-A-2019-9062/interna.md 
---
title: 'Temario GSI (interna)'
Status: hidden
summary: '[BOE-A-2019-9062](https://boe.es/diario_boe/txt.php?id=BOE-A-2019-9062) - Anexo X'
---
$ head -n 5 content/posts/temario/BOE-A-2019-9062/libre.md 
---
title: 'Temario GSI (libre)'
Status: hidden
summary: '[BOE-A-2019-9062](https://boe.es/diario_boe/txt.php?id=BOE-A-2019-9062) - Anexo IX'
---
$ make publish 
pelican /home/u/wks/gsi/content -o /home/u/wks/gsi/output -s /home/u/wks/gsi/publishconf.py 
ERROR: Unknown status 'hidden' for file /home/u/wks/gsi/content/posts/temario/BOE-A-2019-9062/interna.md, skipping it.
ERROR: Unknown status 'hidden' for file /home/u/wks/gsi/content/posts/temario/BOE-A-2019-9062/libre.md, skipping it.
```

**Comment by avaris** [comment]
It's because this feature was added after 4.6.0 and not released yet. Docs for `latest` on RTD refers to the current `master` and may include unreleased features. If you switch to stable (bottom-left arrow), you won't see that section. I know, it is a bit confusing since `latest` also shows version `4.6.0`.

**Comment by justinmayer** [comment]
Hidden status for articles is now supported in the just-released [Pelican 4.7](https://blog.getpelican.com/pelican-4.7-released.html).
